topics/galaxy-ui/tutorials/jupyterlab/tutorial.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: tutorial_hands_on
+
+title: JupyterLab in Galaxy
+zenodo_link: ""
+questions:
+- How can I manipulate data using JupyterLab in Galaxy? 
+- How can I start a notebook in JupyterLab?
+- How can I import/export dataset from/to my history to/from the notebook?
+- How can I save my notebook to my history?
+objectives:
+- "Launch JupyterLab in Galaxy"
+- "Start a notebook"
+- "Import libraries"
+- "Use get() to import datasets from your history to the notebook"
+- "Use put() to export datasets from the notebook to your history"
+- "Save your notebook into your history"
+follow_up_training:
+-
+    type: "internal"
+    topic_name: introduction
+    tutorials:
+        - r-basics
```

Comment:
is this still valid?

Reply:
Yes. It makes little sense to have a follow up tutorial for R basics only. I guess in the future, we could have follow up tutorials for each supported kernels but for now I remove it.